examples/raspberrypi/rp2xxx/src/spi_master.zig

```diff
@@ -0,0 +1,42 @@
+const std = @import("std");
+const microzig = @import("microzig");
+
+const rp2xxx = microzig.hal;
+const time = rp2xxx.time;
+const gpio = rp2xxx.gpio;
+
+const BUF_LEN = 0x100;
+const spi = rp2xxx.spi.instance.SPI0;
+
+// These may change depending on which GPIO pins you have your SPI device routed to.
+const CS_PIN = 17;
+const SCK_PIN = 18;
+// NOTE: rp2xxx doesn't label pins as MOSI/MISO. Instead a pin is always for
+// either receiving or transmitting SPI data, no matter whether the chip is in
+// master or slave mode.
+const RX_PIN = 16;
+
+// Communicate with another RP2040 over spi
+pub fn main() !void {
+    // Set pin functions for CS, SCK, RX
+    const csn = rp2xxx.gpio.num(CS_PIN);
```

Comment:
I noticed that a rp2040 slave does NOT like when the master manually manages the CS pin, so I actually set its function here (vs. the old example)